DoSomething/gambit-conversations
# Removes POST /v1/import-message

#### What's this PR do?
Removes `POST /v1/import-message` route and related configs (e.g. toggle for `GET /v1/broadcasts/:id` webhook settings)

#### How should this be reviewed?
* Verify `GET /v1/broadcasts/:id` works as expected (returns `url` with `customerio-gambit-broadcast`, `body` with `northstarId` and `broadcastId` properties)

* Verify outbound messages work as expected (Broadcast, Front, Signup). No real changes here except removing the Message Outbound config `shouldPostToPlatform` property -- #293 introduces change to only post to platform for SMS Conversations.

* Verify `POST /v1/import-message` returns in a 404

#### Any background context you want to provide?
TODO:

* Update [wiki](https://github.com/dosomething/gambit-conversations/wiki) with v2 endpoints
* Deprecate Blink `customerio-sms-broadcast` resource

#### Relevant tickets
Closes first TODO in #293 

#### Checklist
